src/autogpt_plugins/speech_to_text/speech_to_text_plugin/speech_to_text_plugin.py

```diff
@@ -0,0 +1,52 @@
+import os
+import pyaudio
+import io
+from google.cloud import speech_v1p1beta1 as speech
+from google.cloud.speech_v1p1beta1 import enums
+from google.cloud.speech_v1p1beta1 import types
+import autogpt
+
+os.environ['GOOGLE_APPLICATION_CREDENTIALS'] = 'path/to/credentials.json'
```

Comment:
This line should be fixed up a bit. Look at other examples in the repo of how we read things in